### DISCUSSION TITLE
Unable to admin login getting 404 not found error [Ubuntu 20.04]

### DISCUSSION THREAD

**Discussion by vcgrossi** [microweber/microweber#996] [Q&A]
Hi, i'm new on microweber. 

In my first install, i'm can't acess to admin page.

page:
 http://janussolution.com.br/microweber/admin

PWD: ./microweber/config/microweber.php

<?php return array (
  'version' => '1.3.2',
  'install_default_template' => 'default',
  'install_default_template_content' => 1,
  'compile_assets' => 1,
  'disable_model_cache' => 0,
  'admin_url' => 'admin',
  'site_lang' => 'pt_BR',
  'has_admin' => 1,
  'is_installed' => 1,
  'updated_at' => '2023-02-15 16:55:23',
);
?>

![Capturar](https://user-images.githubusercontent.com/29635334/219115746-a0755920-478d-4de5-bd67-21b0cd7b507a.PNG)

**Comment by markcjf** [comment]
It maybe that you are trying to access a non secure page - add a certificate to your site or if you have try https:// rather than http://
